app/main.py

```diff
@@ -1,8 +1,60 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self: Any, distance_from_city_center: int, clean_power: int,
+            average_rating: float, count_of_ratings:
+            int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        result = 0
+        # __________________________if there are no cars
+        if len(cars) < 1:
+            return 0
+        # __________________________compliance check on clean_power
+        for mark in cars:
+            if mark.clean_mark < self.clean_power:
+                result += self.wash_single_car(mark)
+                mark.clean_mark = self.clean_power
+
+        return round(result, 1)
+
+        # __________________________wash single car
+    def wash_single_car(self, mark: Any) -> float:
+
+        result_v2 = round(self.calculate_washing_price(mark), 1)
+        mark.clean_mark = self.clean_power
+        return result_v2
```

Comment:
wash_single_car - method, that washes a single car, so it should have clean_mark equals wash station's clean_power, if wash_station.clean_power is greater than car.clean_mark
Noting said about calculate_washing_price